LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -267,10 +267,9 @@ public extension PumpManager {
         }
     }
 
-    func enactBolus(units: Double, activationType: BolusActivationType) async throws
-    {
+    func enactBolus(decisionId: UUID?, units: Double, activationType: BolusActivationType) async throws {
```

Comment:
Don't we also need a decisionId on `enactTempBasal`?

Reply:
Does the pump manager need to know the `decisionId`?

Reply:
I think you will need to associate temp basals with dosing decisions, if you are going to display directionality of the dosing decision for temp basal doses. You could try to line up all the historic settings, and override settings and override enactments records and calculate at the time of the dose what it should have been, but the dosing decision is probably more reliable and easy. We may want to update the `AutomaticDoseRecommendation` to just have a directionality enum, too, to simplify things, because right now I think you'd need to do a good amount of fetching and associating to calculate that.